packages/components/modal/src/ModalHeader/ModalHeader.tsx

```diff
@@ -48,6 +50,7 @@ export const ModalHeader = ({
           </Text>
         )}
       </Subheading>
+      {children}
```

Comment:
Do we need to account for any styling/alignment? What should be the default behavior (vertically centered with the rest)?

Reply:
The parent has justify-content: space between, so it should have equal spacing between the title/subtitle & close button. That won't necessarily be centred if the title is particularly long. 

I considered applying a max width to other elements or refactoring to a grid perhaps, but figured that would have potentially breaking changes for existing usage. I reckon this is a reasonable trade-off, WDYT?